src/ui/calendar-panel.zig

```diff
@@ -0,0 +1,40 @@
+const vaxis = @import("vaxis");
+const colors = @import("../styles/colors.zig");
+
+const dummyCalendar = @embedFile("../assets/dummy-calendar.txt");
+
+pub const CalendarPanelDrawOpts = struct {
+    isActivePanel: bool = false,
+};
+
+pub const CalendarPanel = struct {
+    vxWin: *vaxis.Window,
+
+    pub fn init(vxWin: *vaxis.Window) !CalendarPanel {
+        return .{
+            .vxWin = vxWin,
+        };
+    }
+
+    pub fn deinit(self: *const CalendarPanel) void {
+        _ = self;
+    }
+
+    pub fn draw(self: *const CalendarPanel, opts: CalendarPanelDrawOpts) void {
+        const calendarPanelBorderColor = if (opts.isActivePanel)
+            colors.activeBorderColor
+        else
+            colors.borderColor;
+
+        const window = self.vxWin.child(.{
+            .x_off = (self.vxWin.width / 2) + 1,
+            .y_off = 0,
+            .width = .{ .limit = self.vxWin.width / 2 },
+            .border = .{
+                .where = .all,
+                .style = .{ .fg = calendarPanelBorderColor },
+            },
+        });
```

Comment:
The ChildOptions struct thats the input of the vxWin.child function has fields that are usize. This makes it so the potential floats are being cast to usize. We should floor them but for now I didnt do that. 

```suggestion
        const x_off: usize = @intFromFloat((self.vxWin.width / 2) + 1);
        const width: usize = @intFromFloat(self.vxWin.width / 2);

        const window = self.vxWin.child(.{
            .x_off = x_off,
            .y_off = 0,
            .width = .{ .limit = width },
            .border = .{
                .where = .all,
                .style = .{ .fg = calendarPanelBorderColor },
            },
        });
```